Makefile

```diff
@@ -283,7 +283,7 @@ build: $(PILOT_GO_BINS) $(MIXER_GO_BINS) $(SECURITY_GO_BINS)
 # The first block is for aliases that are the same as the actual binary,
 # while the ones that follow need slight adjustments to their names.
 
-IDENTITY_ALIAS_LIST:=istioctl mixc mixs pilot-agent servicegraph sidecar-injector
+IDENTITY_ALIAS_LIST:=istioctl mixc mixs pilot-agent servicegraph sidecar-injector multicluster-ca
```

Comment:
The name here will need to be multicluster_ca (to match the ${ISTIO_OUT}/multicluster_ca) rather than multicluster-ca.  With the way things are currently you'd be able to run "make multicluster-ca" but make would immediately complain that it doesn't know how to make ${ISTIO_OUT}/multicluster-ca.

Reply:
Ah, you are right. I will change the binary to multicluster-ca as well, in a separate PR.